x-pack/plugin/security/qa/service-account/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/service/ServiceAccountIT.java

```diff
@@ -118,6 +118,8 @@ public class ServiceAccountIT extends ESRestTestCase {
               "indices": [
                 {
                   "names": [
+                    "logs",
+                    "logs.*",
```

Comment:
Seeing that we also have `metrics-*` and `traces-*` here, should we also add `metrics`, `metrics.*`, `traces` and `traces.*`?

Reply:
It seems we only need for logs for now: https://github.com/elastic/kibana/issues/224200#issuecomment-2987249888

Reply:
If we keep using the reroute processor I guess it is a matter of time to use it with metrics too. But ok, we can add it when this happens.